doc/user_guide/large_datasets.rst

```diff
@@ -334,10 +334,34 @@ summary statistics to Altair instead of the full dataset.
 
 VegaFusion
 ~~~~~~~~~~
-If you work with large datasets and require your charts to be interactive,
-make sure to check out the `VegaFusion`_ package. It provides serverside acceleration
-for Altair charts. See its documentation on how to use it as well as the current limitations.
+`VegaFusion`_ is a third-party package that re-implements most Vega-Lite transforms for evaluation
+in the Python kernel.  This makes it possible to scale many Altair charts to millions of rows as long as
+they include some form of aggregation.
+
+VegaFusion 1.0 provides two rendering modes that are useful in different situations.
+
+Mime renderer
+^^^^^^^^^^^^^
+The `VegaFusion mime renderer`_ automates the preaggregate and filter workflow described above. Right before
```

Comment:
How about moving the explanation on what it is useful for to the top of these sections? Seems more important to know what it is for before diving into how it works.

```suggestion
The `VegaFusion mime renderer`_ is a good choice for charts that do not re-aggregate or re-filter data in response to selections. It automates the workflow described in :ref:`preaggregate-and-filter`. Right before
```
Could we link to the correct section? In addition to the suggestion above, you'd need to add the reference before the "Preaggregate and Filter in Pandas" header so that it looks like this

```
.. _preaggregate-and-filter:

Preaggregate and Filter in Pandas
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
After the first sentence, we could even include a small code snippet just to show how easy it is to use?

```python
import vegafusion as vf
vf.enable()
```